### PR TITLE
CM-30405 - Add building of executable CLI in the onedir mode and ARM macOS

### PR DESCRIPTION
**Pull request by MarshalX**

### CODE REVIEW

File: poetry.lock [hunk 1]
```diff
@@ -92,13 +92,13 @@ uvloop = ["uvloop (>=0.15.2)"]
 
 [[package]]
 name = "certifi"
```
[the remark this answers is not in the document]
Comment (MarshalX) [reply poetry.lock:94]: #cycode_ignore_non_permissive_license_use mpl is fine

**Comment by MarshalX** [comment]
#cycode_unfail_me

**Comment by MarshalX** [comment]
#cycode_unfail_me

File: poetry.lock (hunk 1 above)
[the remark this answers is not in the document]
Comment (MarshalX) [reply poetry.lock:94]: #cycode_ignore_non_permissive_license_use

[the remark this answers is not in the document]
Comment (MarshalX) [reply poetry.lock:94]: #cycode_ignore_non_permissive_license_use

**Comment by MarshalX** [comment]
#cycode_unfail_me

**Comment by MichalBor** [comment]
#cycode_unfail_me

**Comment by MichalBor** [comment]
#cycode_unfail_me

**Comment by MichalBor** [comment]
#cycode_unfail_me

**Pull request closed without merging**